packages/addon/src/addon.ts

```
@@ -913,12 +928,242 @@ builder.defineStreamHandler(
+              if (aMB < bMB) return 1;
+            }
+
+            return 0;
+          };
+
+          // Apply sorting based on user preference
+          switch (sortingPreference) {
+            case 'size_first':
+              // Size first, then quality, then language
+              const sizeCompare = compareSize();
+              if (sizeCompare !== 0) {
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)